containers/images/pulp/container-assets/pulp-common-entrypoint.sh

```diff
@@ -7,7 +7,7 @@
 # can change our scripts' locations in the future, and call special logic in this
 # script based solely on theo command name.
 
-if [[ "$@" = "pulp-content" || "$@" = "pulp-api" || "$@" = "pulp-worker" || "$@" = "pulp-resource-manager" ]]; then
+if [[ "$@" = "pulp-content" || "$@" = "pulp-api" || "$@" = "pulp-worker" || "$@" = "pulp-resource-manager"  || "$@" = "nginx" ]]; then
```

Comment:
FYI, we can get rid of this entire block

Because /usr/local/bin/pulp-content no longer exists.

Reply:
It gets placed in the container:
https://github.com/pulp/pulp-operator/blob/main/containers/images/pulp/Dockerfile.j2#L83

https://github.com/pulp/pulp-operator/blob/main/containers/images/pulp/container-assets/pulp-content

Should it no longer be there?